applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1373,6 +1373,65 @@ int unicast_client_config_get(struct bt_bap_stream *stream, uint32_t *bitrate,
 	return 0;
 }
 
+static bool sink_locations_get(struct server_store *server, void *user_data)
```

Comment:
Suggest adding a small line of doc, as these foreach functions are of an uncommon nature.